tests/experimental/datasources/test_pandas_datasource.py

```
@@ -48,7 +54,7 @@ def test_construct_csv_asset_directly(csv_path):
     asset = CSVAsset(
         name="csv_asset",
         path=csv_path,
-        regex=r"yellow_tripdata_sample_(\d{4})-(\d{2}).csv",
+        regex=cast(Pattern, r"yellow_tripdata_sample_(\d{4})-(\d{2}).csv"),
```

I don't think we should do this.

---

It eliminates the PyCharm error -- should we just ignore the error?

---

Yes, I think the correct thing to do here is to ignore it with a comment explaining why.

---

@Kilo59 I am adding the comment in next commit.